acceptance/framework/consul/helm_cluster.go

```diff
@@ -297,12 +298,51 @@ func (h *HelmCluster) CreatePortForwardTunnelToResourcePort(t *testing.T, resour
 		require.NoError(r, tunnel.ForwardPortE(t))
 	})
 
+	doneChan := make(chan bool)
+
 	t.Cleanup(func() {
-		tunnel.Close()
+		close(doneChan)
 	})
 
+	go h.monitorPortForwardedServer(t, localPort, tunnel, doneChan, resourceName, remotePort)
+
 	return fmt.Sprintf("127.0.0.1:%d", localPort)
+}
 
+func (h *HelmCluster) monitorPortForwardedServer(t *testing.T, port int, tunnel *terratestk8s.Tunnel, doneChan chan bool, resourceName string, remotePort int) {
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
Do you need a defer ticker.Stop() somewhere?

Reply:
great catch! we absolutely do!